pkg/network/kubemacpool.go

```diff
@@ -88,7 +89,10 @@ func fillDefaultsKubeMacPool(conf, previous *opv1alpha1.NetworkAddonsConfigSpec)
 
 func changeSafeKubeMacPool(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []error {
 	if prev.KubeMacPool != nil && !reflect.DeepEqual(prev.KubeMacPool, next.KubeMacPool) {
```

Comment:
`if prev.KubeMacPool != nil && next.KubeMacPool != nil &7 !equal`, no need for the second if

Reply:
what does &7 !equal means ?

you mean just to change to 
if prev.KubeMacPool != nil && next.KubeMacPool != nil 
?
if so i dont understand, we allow the following
prev.KubeMacPool != nil && next.KubeMacPool == nil

guess i miss something

Reply:
IF kubemacpool was set AND kubemacpool should be still set AND the previous configuration is not the same as the future one; THEN and only then raise and error.